modules/compliance-dashboard-overview.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-compliance.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="compliance-dashboard-overview_{context}"]
+= Compliance dashboard overview
+
+After you have performed a compliance scan, the compliance dashboard displays the results as the compliance status for your environment. You can view compliance violations directly from the dashboard, filter the detailed view and drill down into the compliance standards to find out if your environment is compliant against specific benchmarks. 
```

Comment:
Split out the last sentence: "You can view compliance violations directly from the dashboard. To find out if your environment is compliant against specific benchmarks, filter the detailed view and drill down into the compliance standards."